pages/language/ref/cells.mdx

```diff
@@ -16,254 +16,316 @@ Creates a new empty Builder.
 extends fun endCell(self: Builder): Cell;
 ```
 
-Converts `Builder` into an ordinary `Cell`.
+Extension function for the Builder.
+
+Converts a Builder into an ordinary Cell.
 
 ## Builder.storeUint
 
 ```tact
 extends fun storeUint(self: Builder, value: Int, bits: Int): Builder;
 ```
 
-Stores an unsigned `bits`-bit integer `value` into `s` for `0 ≤ bits ≤ 256`.
+Extension function for the Builder.
+
+Stores an unsigned `bits`-bit Int `value` into the Builder for $0 ≤ bits ≤ 256$. Returns modified Builder.
 
 ## Builder.storeInt
 
 ```tact
 extends fun storeInt(self: Builder, value: Int, bits: Int): Builder;
 ```
 
-Stores a signed `len`-bit integer `x` into `b` for `0 ≤ len ≤ 257`.
+Extension function for the Builder.
+
+Stores a signed `bits`-bit Int `value` into the Builder for $0 ≤ bits ≤ 257$. Returns modified Builder.
 
 ## Builder.storeBool
 
 ```tact
 extends fun storeBool(self: Builder, value: Bool): Builder;
 ```
 
-Stores Bool `value` into Builder `s`. It will write to `s` integer `x`. `x = -1` if `value = True` or `x = 0` integer if `value = False`.
+Extension function for the Builder.
+
+Stores a Bool `value` into the Builder. Writes `-1` if `value` is `true`, and writes `0` otherwise. Returns modified Builder.
 
 ## Builder.storeSlice
 
 ```tact
 extends fun storeSlice(self: Builder, cell: Slice): Builder;
 ```
 
-Stores slice `cell` into builder `s`.
+Extension function for the Builder.
+
+Stores a Slice `cell` into the Builder. Returns modified Builder.
 
 ## Builder.storeCoins
 
 ```tact
 extends fun storeCoins(self: Builder, value: Int): Builder;
 ```
 
-Stores (serializes) an integer `value` in the range `0..2^120 − 1` into builder `s`. The serialization of `value` consists of a 4-bit unsigned big-endian integer `l`, which is the smallest integer `l ≥ 0`, such that `value < 2^8l`, followed by an `8l`-bit unsigned big-endian representation of `value`. If `value` does not belong to the supported range, a range check exception is thrown.
+Extension function for the Builder.
+
+Stores (serializes) an integer `value` in the range $0 .. 2^{120} − 1$ into the Builder. The serialization of `value` consists of a 4-bit unsigned big-endian integer $l$, which is the smallest integer $l ≥ 0$, such that `value` $< 2^{8 * l}$, followed by an $8 * l$-bit unsigned big-endian representation of `value`. If `value` does not belong to the supported range, a range check exception is thrown. Returns modified Builder.
```

Comment:
Could we link here to some page in the TON docs that explains weird numbers like 120 and some other stuff?